custom-checks/checkstyle/pom.xml

```diff
@@ -63,12 +80,6 @@
       <artifactId>org.eclipse.jdt.annotation</artifactId>
       <version>${jdt-annotations.version}</version>
     </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-      <version>${slf4j.version}</version>
-      <scope>provided</scope>
```

Comment:
The old dependency was with scope "provided" - is it intended that this has been removed with the upgrade?

Reply:
let me check if provided works here as well...

Reply:
Doesn't it?